Resources/Prototypes/_RMC14/Roles/Jobs/Other/senior_enlisted_advisor.yml

```
@@ -20,6 +20,24 @@
   - !type:DepartmentTimeRequirement
     department: CMCommand
     time: 18000 # 5 hours
+  ranks:
+    RMCRankSergeantMajor:
+    - !type:RoleTimeRequirement
+      role: CMJobSeniorEnlistedAdvisor
+      time: 630000 # 175 hours
+    RMCRankMasterGunnerySergeant:
+    - !type:RoleTimeRequirement
+      role: CMJobSeniorEnlistedAdvisor
+      time: 252000 # 70 hours
+    RMCRankFirstSergeant:
+    - !type:RoleTimeRequirement
+      role: CMJobSeniorEnlistedAdvisor
+      time: 90000 # 25 hours
+    RMCRankMasterSergeant:
+    - !type:RoleTimeRequirement
+      role: CMJobSeniorEnlistedAdvisor
+      time: 36000 # 10 hours
+    RMCRankGunnerySergeant: []
```

This has since been changed, it's now only gunnery sergeant at all playtimes
```suggestion
  ranks:
    RMCRankGunnerySergeant: []
```